addons/website_sale/views/templates.xml

```diff
@@ -1764,4 +1764,21 @@
           </div>
         </t>
     </template>
+
+    <!-- When open cart from abandoned email template and already existing diffrent cart then link in session, Ask for replace/merge the existing cart with the new cart -->
+    <template id="abandoned_cart_restore" inherit_id="website_sale.cart" name="abandoned cart restore">
```

Comment:
Probably because this whole template file is constructed the same way. Personally I find it easier/less messy when it's separated.